pkg/graphql/graphql.go

```diff
@@ -1566,11 +1566,12 @@ type GetTemplateVariables struct {
 
 type GetTemplateResponse struct {
 	Template struct {
-		ID          string `json:"id"`
-		Name        string `json:"name"`
-		Description string `json:"description"`
-		Provider    string `json:"provider"`
-		State       struct {
+		ID             string `json:"id"`
+		Name           string `json:"name"`
+		DistributionId string `json:"distribution_id"`
```

Comment:
```suggestion
		DistributionId string `json:"distributionId"`
```

IMHO it should be `distributionId`

Reply:
Yep